runtime/compiler/runtime/SignalHandler.c

```diff
@@ -1850,6 +1850,87 @@ UDATA jitAMD64Handler(J9VMThread* vmThread, U_32 sigType, void *sigInfo)
 	return J9PORT_SIG_EXCEPTION_CONTINUE_SEARCH;
 }
 
+#elif defined(TR_HOST_ARM64) && defined(TR_TARGET_ARM64)
+
+extern void jitHandleNullPointerExceptionTrap(void);
+extern void jitHandleInternalErrorTrap(void);
+
+UDATA jitARM64Handler(J9VMThread* vmThread, U_32 sigType, void* sigInfo)
+{
+	PORT_ACCESS_FROM_VMC(vmThread);
+
+	J9JITExceptionTable *exceptionTable = NULL;
+	J9JITConfig *jitConfig = vmThread->javaVM->jitConfig;
+
+	if (jitConfig) {
+		const char* infoName;
```

Comment:
Mixed styles of declaring pointer variables.
`const char* infoName;` or `const char *infoName;`

Reply:
Fixed